controllers/comments.controller.js

```diff
@@ -17,3 +17,14 @@ exports.postCommentByReviewId = (req, res, next) => {
     .catch(next)
 }
 
+exports.deleteCommentById = (req, res, next) => {
+   
+    const { comment_id } = req.params; 
+
+    return removeCommentById(comment_id).then(() => {
+        res.status(204).send()
```

Comment:
You can clean this up with `res.sendStatus(204)`

Then you dont need an empty send